validation_pipeline.ipynb

```diff
@@ -25,12 +25,14 @@
         "DEBUG = False #@param {type:\"boolean\"}\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    fake_sample = dataset.create_fraudulent_samples(
</pre><p>Is this just for Oxygen or for everything?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
It's for everything. Updated to reflect this.